src/frontend/templates/base.html

```diff
@@ -51,23 +51,33 @@
           </a>
         </div>
 
+        <div class="navbar-end">
+          <a class="navbar-item is-active" href="https://github.com/lightvector/KataGo/releases">
+            {% trans "Releases" %}
+          </a>
+          <a class="navbar-item is-active" href="https://github.com/lightvector/KataGo">
+            {% trans "GitHub (KataGo)" %}
+          </a>
+          <a class="navbar-item is-active" href="https://github.com/katago/katago-server">
+            {% trans "GitHub (Webserver)" %}
+          </a>
+
```

Comment:
nice.

is-active vs is-hidden-desktop-only ?

Reply:
I don't know what is-active actually does, I only did it to mimic the existing menu items. :)